test/provision/terraform/main.tf

```diff
@@ -37,10 +34,27 @@ variable "inventory_output" {
   default = "./inventory.ec2"
 }
 
+variable "windows_ec2" {
+  default = ""
+}
+
+variable "linux_ec2" {
+  default = ""
+}
+
+variable "ec2_prefix" {
+  default = ""
+}
+
+locals {
+    filtered_ec2 = var.platform == "windows" ? var.windows_ec2 : var.linux_ec2
+}
+
 module "env-provisioner" {
-  source             = "git::https://github.com/newrelic-experimental/env-provisioner//terraform/otel-ec2"
-  ec2_otels         =    {for key, val in var.ec2_infra_agents:
-                            key => val if val.platform == var.platform || var.platform == "all"}
+  source             = "git::https://github.com/newrelic-experimental/env-provisioner//terraform/otel-ec2?ref=NR-164382_add_windows_instances"
```

Comment:
Remove testing branch ref?

```suggestion
  source             = "git::https://github.com/newrelic-experimental/env-provisioner//terraform/otel-ec2"
```

Reply:
Fixed in https://github.com/newrelic/infrastructure-agent/pull/1822/commits/24a6aee6560c3a7a488c837056cc4bde7bd74005